php/libraries/Database.class.inc

```diff
@@ -760,15 +760,15 @@ class Database
     }
 
     /**
-     * Runs a query as a prepared statement and returns the values of the first
-     * column in the select statement. If multiple columns are given, only the first
-     * will be returned.
+     * Runs a query as a prepared statement and returns the values of the
+     * column given in the select statement. If multiple columns are given, an error
+     * is thrown.
      *
      * @param string $query  The SQL SELECT query to be run
      * @param array  $params Values to use for binding to prepared statement
      *
      * @return array Associative array of form rowID=>value containing all values
```

Comment:
although any array (in PHP) is an associative array, since rowID is used, it should be accessible with $array[0], $array[1], ...

Should we remove the "Associative"?

Reply:
I still like the distinction between intentianally associative (i.e. the key is valuable information) vs auto incremented associative... personal opinion

Reply:
I think if someone wants they can make another PR to debate this point, but within the scope of this PR it's already fixing the other code comment, so I'm just going to merge it.